languagetool-language-modules/de/src/main/resources/org/languagetool/resource/de/added.txt

```diff
@@ -3862,3 +3862,147 @@ Get-togethers	Get-together	SUB:NOM:PLU:NEU
 Get-togethers	Get-together	SUB:GEN:PLU:NEU
 Get-togethers	Get-together	SUB:DAT:PLU:NEU
 Get-togethers	Get-together	SUB:AKK:PLU:NEU
+Anleiter	Anleiter	SUB:NOM:SIN:MAS
+Anleiters	Anleiter	SUB:GEN:SIN:MAS
+Anleiter	Anleiter	SUB:DAT:SIN:MAS
+Anleiter	Anleiter	SUB:AKK:SIN:MAS
+Anleiter	Anleiter	SUB:NOM:PLU:MAS
+Anleiter	Anleiter	SUB:GEN:PLU:MAS
+Anleiter	Anleiter	SUB:DAT:PLU:MAS
```

Comment:
-> "Anleitern", oder?

Reply:
Bei Anleiter steht das nicht im Duden, weswegen ich mich noch umgeschaut habe.
Anleiter habe ich nicht direkt gefunden, aber bei Leiter kommt in der Pluralform Anleitern vor.